src/containers/edit-tutor-profile/NotificationContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import Box from '@mui/system/Box'
+import Typography from '@mui/material/Typography'
+import Switch from '@mui/material/Switch'
+
+import { useTranslation } from 'react-i18next'
+
+import SettingItem from '~/components/setting-item/SettingItem'
+import { styles } from '~/containers/edit-tutor-profile/NotificationContainer.style'
+export const NotificationContainer = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={styles.root}>
+      <Typography sx={styles.title}>
+        {t('editTutor.notificationTab.notifications')}
+      </Typography>
+      <Typography sx={styles.subtitle}>
+        {t('editTutor.notificationTab.mainHint')}
+      </Typography>
+
+      <Box sx={styles.optionsContainer}>
+        <SettingItem
```

Comment:
maybe it is better to create an array with each item as an object with the title and subtitle, then, just use map for rendering <SettingItem/>